base/docs/basedocs.jl

```diff
@@ -22,6 +22,7 @@ as well as many great tutorials and learning resources:
 
 For help on a specific function or macro, type `?` followed
 by its name, e.g. `?cos`, or `?@time`, and press enter.
+Type `;` to enter shell mode, `]` to enter package mode.
```

Comment:
I'd suggest adding an "or" before the \`]`

Reply:
I find "or" to make it sound like to options to pick from . You mean that the comma is not separating the two parts enough? In that case maybe a newline between the two?

Reply:
Sure, making them fully independent sentences ~~works~~ would have worked too. For example:

    Type `;` to enter shell mode; type `]` to enter package mode.

With the comma alone, the sentence matches informal speech but is not grammatically correct, IIUC.